host_core/native/hostcore_wasmcloud_native/src/oci.rs

```diff
@@ -1,13 +1,12 @@
+use oci_distribution::manifest::OciManifest;
+use oci_distribution::secrets::RegistryAuth;
+use sha2::{Digest, Sha256};
```

Comment:
Nit: for imports our convention is:
```
[std lib]
// empty line
[3rd party like oci_distribution]
// empty line
[1st party like crate::foo]
```

Reply:
Yes, I saw Taylor's review comments after pushing my commit. My bad. 
The imports were something that clippy/rust fmt recommended, I will change it back